### ISSUE TITLE
Expose oreBalanceToNumber utility function

### ISSUE THREAD

**Issue by mxmnci** [JuanRdBO/ore-sdk#1]
@JuanRdBO , love the work you've done so far on this! 

I noticed that the `oreBalanceToNumber` function isn't exposed in the SDK yet. This function would be particularly useful when using the `getProof` method which returns the staked account balance as a bigint. 

```ts
const result = await ore.getProof();
if (result?.balance) {
    console.log(oreBalanceToNumber(result.balance));
}
 ```

In the meantime, I've copied the constants and `oreBalanceToNumber` function over to the project I'm working on so I can use it in the interim, but I thought I'd give you a heads up

**Comment by JuanRdBO** [comment]
Hey @mxmnci, I fixed this in the latest commit (58751d011e5d27b1f75c7d2b4408c3cd1a9a4228). Test it and lmk!

**Comment by JuanRdBO** [comment]
Oh, forgot to mention it's on version `0.0.2` of the `ore-sdk` package
